proto/consensus/msgs.proto

```diff
@@ -8,15 +8,15 @@ import "proto/types/types.proto";
 import "proto/libs/bits/types.proto";
 
 message NewRoundStep {
-  int64 height                   = 1;
-  int32 round                    = 2;
-  int32 step                     = 3 [(gogoproto.casttype) = "int8"];
-  int64 seconds_since_start_time = 4;
-  int32 last_commit_round        = 5;
+  int64  height                   = 1;
+  int32  round                    = 2;
+  uint32 step                     = 3;
```

Comment:
why not uint8?

Reply:
someone could send a number greater that uint8 over the wire then on unmarshal it would overflow panic.